### DISCUSSION TITLE
How format a numeric value in templates?

### DISCUSSION THREAD

**Discussion by yjaigu** [fbaligand/kibana-enhanced-table#184] [Q&A] [answered]
HI,

I would like to format a column like this : 10,2% (+2,7%)
I computed the second value in another hidden column but I can't manage to format it.
It looks like this : 10,2% (2.6999999997%).

How can I do this ?
Thanks.
Yann

**Comment by fbaligand** [comment, marked as answer]
Hi,

at this time, you cannot reference a previous “formatted” column.
I will try to add this feature in next release.

that said, at this time, in a formula, you can use this function:
roundTo(x, n)
where x is the number to round, and n is the number of decimals you wish.
This is not perfect but it helps.

**Comment by fbaligand** [comment]
New version 1.14.0 is out, with a new feature in computed column template: you can reference another column formatted value.
Example: `{{ formattedCol2 }}`

https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.14.0
